packages/hoprd/testenv.ts

```diff
@@ -0,0 +1,66 @@
+import { validate } from 'jsonschema'
```

Comment:
This file should be in the `test` subfolder and also be named properly. At least some basic documentation on what is test does would be useful.